ska_conda/ska_builder.py

```diff
@@ -85,11 +70,20 @@ def build_one_package(self, name, tag=None):
         self._build_package(name)
 
     def build_list_packages(self):
+        failures = []
         with open(build_list, "r") as f:
             for line in f.readlines():
                 pkg_name = line.strip()
                 if not pkg_name.startswith("#"):
-                    self.build_one_package(pkg_name)
+                    try:
+                        self.build_one_package(pkg_name)
+                    # Just try to build the rest if there's a failure, but record the name
+                    except:
+                        failures.append(pkg_name)
+                        continue
+        if len(failures):
+            raise ValueError("Packages {} failed".format(",".join(failures)))
```

Comment:
This isn't strictly required, but I thought it might be helpful to let the build process continue through the rest of the packages even if you have an issue with one package (an issue such as forgetting your ssh key password) and then printing out the failures and exiting with error status at the end.

Reply:
I sort of think that if something fails, there is really a problem that needs to be fixed before moving on (and maybe you can into dependency hell where every build starts failing because of the missing failed build).  Since it doesn't re-build everything every time (right?), it seems pretty cheap to restart.  The "forgetting ssh key" seems a not-credible failure.